userProfile/privateProfile.mjs

```diff
@@ -29,29 +30,26 @@ router.use(
   })
 )
 
-router.get("/profile", async (req, res) => {
-  const user = await req.user 
-  if (!user) return respondWithError(res, 401, "Unauthorized user") 
+router.get("/profile", auth0Middleware(), async (req, res) => {
+  const user = await req.user
+  if (!user) return respondWithError(res, 401, "Unauthorized user")
   const userObj = new User(user._id)
```

Comment:
You should add a mechanism to these in case userObj.getSelf() fails so that this endpoint can respond RESTfully about what error may have occurred. Without any handling on this, it could cause an incorrect response or even a code crash.